terraform/central-account/data.tf

```diff
@@ -28,21 +28,19 @@ data "template_file" "consoleme_config" {
     sync_accounts_from_organizations_role_to_assume    = var.sync_accounts_from_organizations_role_to_assume
     application_admin                                  = var.application_admin
     region                                             = data.aws_region.current.name
+    jwt_email_key                                      = var.lb-authentication-jwt-email-key
     user_facing_url                                    = "https://${aws_lb.public-to-private-lb.dns_name}:${var.lb_port}"
   }
 }
 
 data "template_file" "consoleme_userdata" {
   template = file("${path.module}/templates/userdata.sh")
   vars = {
-    bucket                  = aws_s3_bucket.consoleme_source_bucket.bucket
+    bucket                  = aws_s3_bucket.consoleme_files_bucket.bucket
     current_account_id      = data.aws_caller_identity.current.account_id
-    demo_config             = data.template_file.consoleme_config.rendered
     region                  = data.aws_region.current.name
-    CONFIG_LOCATION         = var.CONFIG_LOCATION
-    CONSOLEME_CONFIG_S3     = var.CONSOLEME_CONFIG_S3
+    CONFIG_LOCATION         = "/apps/consoleme/example_config/example_config_terraform.yaml"
```

Comment:
Should we make this something standard like "/etc/consoleme/config/config.yaml" ?

Reply:
Actually, we can just remove this variable entirely. "/etc/consoleme/config/config.yaml" is where your new retrieve scripts will put it in, and config.py has this /etc path in its defaults.

Agree?

Reply:
Yup that works too

Reply:
Hmmm:
```
[root@ip-10-1-1-11 bin]# grep "Retrieved ConsoleMe's configuration from" /var/log/cloud-init-output.log  -A 20
2020-12-20 06:43:34,334 - INFO - root - [retrieve_or_decode_configuration.py:78 - <module>() ] - Retrieved ConsoleMe's configuration from s3://your-name-prefix-lrhb/config.yaml and writing  to /etc/consoleme/config/config.yaml
Traceback (most recent call last):
  File "/apps/consoleme/scripts/retrieve_or_decode_configuration.py", line 81, in <module>
    make_directories(save_location)
  File "/apps/consoleme/scripts/retrieve_or_decode_configuration.py", line 44, in make_directories
    os.makedirs(head)
  File "/usr/lib64/python3.8/os.py", line 213, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/usr/lib64/python3.8/os.py", line 223, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/etc/consoleme'
```

Reply:
@castrapel can you fix this separately, then I'll merge it into my branch?

Reply:
Sure that sounds good